Dockerfile

```diff
@@ -132,8 +133,8 @@ EXPOSE 80 443
 
 LABEL name="manageiq" \
           vendor="ManageIQ" \
-          version="Capablanca" \
-          release="latest" \
+          version="Darga" \
+          release=${REF} \
```

Comment:
I assume the release needs to match the docker tag which is not the REF.  e.g. master => latest, but darga => latest-darga